source/mathPres/__init__.py

```diff
@@ -31,7 +33,7 @@ def getSpeechForMathMl(self, mathMl):
 		@param mathMl: The MathML markup.
 		@type mathMl: str
 		@return: A speech sequence.
-		@rtype: list of str and/or L{speech.SpeechCommand}
+		@rtype: List[str, speech.SpeechCommand]

```

Comment:
This can now be type annotated I assume

Reply:
No, I don't think so.  Because speech imports mathPres. Hopefully we eventually move things around so we stop running into these issues.